objects/src/transaction/outputs.rs

```diff
@@ -206,100 +136,3 @@ fn build_output_notes_commitment<T: ToEnvelope>(notes: &[T]) -> Digest {
 
     Hasher::hash_elements(&elements)
 }
-
-// OUTPUT NOTE
-// ================================================================================================
-
-/// An note create during a transaction.
-///
-/// When a note is produced in a transaction, the note's recipient, assets, and metadata must be
-/// known. However, other information about the note may or may not be know to the note's producer.
-#[derive(Debug, Clone, PartialEq, Eq)]
-pub struct OutputNote {
```

Comment:
Remove in favor of an enum of `NoteConsumable` and `NoteEnvelope`, which correspond to public and private notes.